src/Entity/Unit.php

```diff
@@ -92,7 +92,7 @@ public function setNumber($number)
      */
     public function setValue($value)
     {
-        $this->value = number_format($value, 6);
+        $this->value = number_format($value, 6, '.', '');
```

Comment:
This change here is because the number_format() call automatically pops in a thousands comma separator. The UPS Shipping API considers any values > 999.99 to be invalid because "," isn't a valid character in the <Unit><Value></Value></Unit> node.